code-challenges.test.js

```diff
@@ -14,39 +14,135 @@
+// return firstLastNameCapital outside of the HOF block but within setenceWithFirstLastCapitalized function.
 
 // --------------------2) Create a function that takes in a mixed data array and returns an array of only the REMAINDERS of the numbers when divided by 3.
 
 // a) Create a test with an expect statement using the variables provided.
 
-const hodgepodge1 = [23, "Heyyyy!", 45, -10, 0, "Yo", false]
-// Expected output: [ 2, 0, -1, 0 ]
-const hodgepodge2 = [5, "Hola", 43, -34, "greetings", true]
-// Expected output: [ 2, 1, -1 ]
+describe("onlyRemainders", () => {
+  it("returns an array of only the REMAINDERS of the numbers when divided by 3.", () => {
+    const hodgepodge1 = [23, "Heyyyy!", 45, -10, 0, "Yo", false]
+    // Expected output: [ 2, 0, -1, 0 ]
+    const hodgepodge2 = [5, "Hola", 43, -34, "greetings", true]
+    // Expected output: [ 2, 1, -1 ]
+    
+    expect(onlyRemainders(hodgepodge1)).toEqual([ 2, 0, -1, 0 ])
+    expect(onlyRemainders(hodgepodge2)).toEqual([ 2, 1, -1 ])
+  })
+})
+
+// ReferenceError: onlyRemainders is not defined
 
 // b) Create the function that makes the test pass.
 
+const onlyRemainders = (param) => {
+  // Use HOF .filter() to iterate and filter out everything that is not a number.
+  return param.filter(value => typeof value  === "number")
+  // use HOF .map() to interate each value with modulo by 3 to get the remainders.
+  .map(value => value % 3)
+}
+
 // Pseudo code:
+// Create a function called onlyRemainders. Takes 1 parameter.
+// Use HOF .filter to iterate over array to check for only Numbers.
+// use built-in method typeof, strict equailty operator, and 'number' to specify the datatype.
+// use .map and modulo 3 to get the remainders.
+
+// Test Suites: 1 passed, 1 total
+// Tests:       2 passed, 2 total
 
 // --------------------3) Create a function that takes in an array of numbers and returns the sum of all the numbers cubed.
 
 // a) Create a test with an expect statement using the variables provided.
 
-const cubeAndSum1 = [2, 3, 4]
-// Expected output: 99
-const cubeAndSum2 = [0, 5, 10]
-// Expected output: 1125
+describe("sumOfCubed", () => {
+  it("returns the sum of all the numbers cubed.", () => {
+    const cubeAndSum1 = [2, 3, 4]
+    // Expected output: 99
+    const cubeAndSum2 = [0, 5, 10]
+    // Expected output: 1125
+
+    expect(sumOfCubed(cubeAndSum1)).toEqual(99)
+    expect(sumOfCubed(cubeAndSum2)).toEqual(1125)
+  })
+})
+
 
 // b) Create the function that makes the test pass.
+const sumOfCubed = (param) => {
+  // declare a new variable, use HOF .map() to iterate on parameter, use ** to indicate a exponent.
+  let cubedOfNums = param.map(value => value **3)
+  // use HOF .reduce() to iterate over cubedOfNums to accumlate a single value. Using two params.
+  return cubedOfNums.reduce((acc, currentValue) => 
```

Comment:
Nice! Quick note, you can also refactor using just .reduce:
`return param.reduce((acc, num) => acc + num ** 3, 0)`
Just saves a line of code!